README.md

```diff
@@ -133,6 +133,8 @@ You should see a JSON response like that below:
 }
 ```
 
+Comparing to the original [sample email](samples/sample1.json), the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset. 
```

Comment:
```suggestion
The text summarizer preserves in the summary text some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc. that appear in the input [sample](samples/sample1.json), which is excerpted from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset
```

Reply:
Thanks @xuhdev.  Hi @MLnick, may I also have your suggestions? Thanks.

Reply:
This seems fine